src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderMap.cs

```diff
@@ -26,16 +35,35 @@ private CoseHeaderMap(bool isReadOnly)
 
         private ICollection<KeyValuePair<CoseHeaderLabel, CoseHeaderValue>> HeaderParametersAsCollection => _headerParameters;
 
+        /// <summary>
+        /// Gets a collection containing the labels in the <see cref="CoseHeaderMap"/>.
+        /// </summary>
         public ICollection<CoseHeaderLabel> Keys => _headerParameters.Keys;
 
+        /// <summary>
+        /// Gets a collection containing the values in the <see cref="CoseHeaderMap"/>.
+        /// </summary>
         public ICollection<CoseHeaderValue> Values => _headerParameters.Values;
 
+        /// <summary>
+        ///   Gets the number of label/value pairs contained in the <see cref="CoseHeaderMap"/>.
+        /// </summary>
         public int Count => _headerParameters.Count;
 
+        /// <inheritdoc/>
         IEnumerable<CoseHeaderLabel> IReadOnlyDictionary<CoseHeaderLabel, CoseHeaderValue>.Keys => _headerParameters.Keys;
 
+        /// <inheritdoc/>
         IEnumerable<CoseHeaderValue> IReadOnlyDictionary<CoseHeaderLabel, CoseHeaderValue>.Values => _headerParameters.Values;
 
+        /// <summary>
+        /// Gets or sets the value associated with the specified label.
+        /// </summary>
+        /// <param name="key">The label of the value to get or set.</param>
+        /// <returns>The value associated with the specified label.</returns>
```

Comment:
Indexers are special kinds of properties, rather than special kinds of methods, they don't have returns statements.

```suggestion
```

Reply:
huh, when you type `///` on top of the indexer it generates a `<summary>`, `<param name="key">` and `<returns>`. cc @carlossanlop.

Reply:
Was this indexer recently added? I was trying to find the structure of this API in dotnet-api-docs to determine what triple slash elements are needed, but I can't find it in either of the two docs files for this type:

https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Security.Cryptography.Cose/CoseHeaderMap.xml

https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Security.Cryptography.Cose/CoseHeaderMap%2BEnumerator.xml

@gewarren do you know if indexers show up in MS Docs?

Reply:
It is new API in 7.0. You can check Dictionary's indexer.